index: 20210312.1 Hackathon Nayarit 2021

xaur opened the pull request:
PREVIEW: https://github.com/decredcommunity/events/blob/naya/index/20210312.1.yml

xaur committed:
index: add 20210312.1

adcadde commented on index/20210312.1.yml:
```diff
@@ -0,0 +1,136 @@
+organizers:
+  - org: Ministry of Education of Nayarit
+    url: https://www.facebook.com/EduNayarit/
+    person: 
```
Elian

xaur replied:
@adcadde I added two organizations as "organizers". Optionally, it is possible to add also mark a certain person from the org. Do you mean to add our Elian to "Decred Latam team" on the next line, or is there another Elian at the "Ministry of Education of Nayarit" org?

adcadde commented on index/20210312.1.yml:
```diff
@@ -0,0 +1,136 @@
+  - https://twitter.com/Decred_ES/status/1366465819329368066
+  - https://twitter.com/Decred_ES/status/1366465867635183617
+  - https://twitter.com/Decred_ES/status/1365496737566179328
+  - https://matrix.to/#/!clHjlICBEtCtAdTupf:decred.org/$9bNnZG8xlVK9xN-k-0IS2opKchaasmuMISGX7xV8l38
```
https://www.facebook.com/EduNayarit/posts/2989483187938352

adcadde committed:
Update 20210312.1.yml

Added the EN titles, descriptions of sub-events, other media sources and changed the attendance field to views with up-to-date counts.

adcadde reviewed (approved):
@xaur the person field fixes this -thanks-, it's good to have the individual agent responsible for the organization of each activity. In this case, it was Elian.

adcadde committed:
Update

Added another announcement source

xaur committed:
fix structure

xaur committed:
cleanup

xaur commented:
Thanks for the additions @adcadde. I changed the `views` field back to `attendance` list because this is what the site generating code expects.

Questions:

1. Were the webinars streamed on Facebook and then uploaded to YouTube and Periscope, or were the streamed to all 3 at once? (this is to correctly set the `location` field)

2. Which platform the view numbers came from, or are these the aggregate numbers across all platforms? (I try to specify what each number means to see which distribution methods work best)

3. We have 5 webinars almost written up, but what about the actual hackathon? Did it happen? Was it one "subevent" or multiple rounds? What were the projects about? It is the goal of the whole series and I think we shall write it up. It would be interesting to know the influence of the webinars, i.e. how people attending them have performed in the hackathon and what projects they worked on.

4. "Decred Contractor Model" is missing description.

adcadde commented:
Hi @xaur!

Answers:

1. All transmitted at the same time using Streamyard, `location` can be more granular and mention 'Facebook, Periscope & Youtube' or just 'Internet'. 
2. Numbers are an aggregate of our social media and the partner's social media. I can specify.
3. I cannot give word about the hackathon other than it was all carried out by the partner. Our participation was to impart the webinars, @3lian could give a better picture of the details.
4. I'll add one.

adcadde committed:
Attendance field & sub-event field update

Numbers of each sm and added description of 5th webinar.

xaur commented:
Thanks for detailed stats!

> All transmitted at the same time using Streamyard, location can be more granular and mention Facebook, Periscope & Youtube' or just 'Internet'.

Good to know you have that set up. And yes I try to be more granular and stick to a certain syntax. It will allow to generate and index page with e.g. all events involving Periscope, if we ever need to analyze that platform.

I'll wait for Elian's comment on the hackathon for the final bit of information before finishing this.

3lian committed:
Update 20210312.1.yml

Update 20210312.1.yml

xaur committed:
fix structure for views

xaur committed:
tweak text

xaur committed:
platforms

xaur commented:
Merged and published: https://decredcommunity.github.io/events/index/20210312.1

Great work guys!
